### ISSUE TITLE
AES Encrypt/Decrypt helper functions (compatible with CryptoJS) needed

### ISSUE THREAD

**Issue by yakuter** [passwall/passwall-server#136]
**Is your feature request related to a problem? Please describe.**
For better security, we need to encrypt and decrypt the json data itself transmitted in rest api communication. 

**Describe the solution you'd like**
[CryptoJS](https://github.com/brix/crypto-js) javascript package is used on frontend for encryption. So we need to decrypt what CryptoJS encrypts on Go (server) side.

By the way, CryptoJS uses CBC mode and Pkcs7 padding as default. 
https://cryptojs.gitbook.io/docs/

Basically we need two helper function (encrypt and decrypt) compatible with CryptoJS to use in API communication .

Does anyone wants to work on this?

**Comment by yusufpapurcu** [comment]
I can do this issue

**Comment by yusufpapurcu** [comment]
Finished waiting for review.

**Comment by yakuter** [comment]
I tried it. Works fine, thank you @yusufpapurcu great job ;)
